test/tests/functional/pbs_pbsnodes.py

```diff
@@ -89,3 +89,21 @@ def test_pbsnodes_aS(self):
             self.assertIn(
                 hdr, out2['out'][0],
                 "header %s not found in output" % hdr)
+
+    def test_pbsnodes_av(self):
+        """
+        This verifies the values of last_used_time in 'pbsnodes -av'
+        result before and after server shutdown, once a job submitted.
+        """
+        j = Job(TEST_USER)
+        j.set_sleep_time(1)
+        jid = self.server.submit(j)
+        self.server.accounting_match("E;%s;" % jid)
+        val_before = self.server.status(NODE)[0]['last_used_time']
```

Comment:
This isn't what I was suggesting.  I was suggesting you do a self.server.status(NODE, 'last_used_time').  The dictionary that is returned has just last_used_time (and id) in it.

Reply:
Doing "self.server.status(NODE,'last_used_time')" returns "[{'last_used_time': 'Mon Oct 29 12:01:26 2018', 'id': 'centosvm'}]". Modified the status querying and parsed the same accordingly.

Is there any advantage of doing "self.server.status(NODE,'last_used_time')[0]['last_used_time']" instead of "self.server.status(NODE)[0]['last_used_time']" ?

Reply:
Thanks @bhroam for the explanation. It will help in future PTL test cases also.